package.json

```diff
@@ -74,7 +74,8 @@
   "license": "MIT",
   "resolutions": {
     "react": "^17.0.0",
-    "react-dom": "^17.0.0"
+    "react-dom": "^17.0.0",
+    "@storybook/react/webpack": "^5"
```

Comment:
Storybook was a huge pain to get working with Webpack 5. They supply these builder and manager, but it has Webpack 4 as a dependency. This causes `yarn` to get confused because it has to choose which webpack to run when we run it via `ui-scripts`, so it needs to be specified here (and in some other places just to be sure :/ ).
Sotrybook plans to remove its Webpack 4 dependency with v6, so then we can remove this code